### PR TITLE
First stab at updating the readme

### PR DESCRIPTION
**Pull request by joshblum**
Open to any comments. @anishathalye any thoughts you have too would be appreciated :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,56 @@
...
-You can pass options in any order, but the last time an option is passed will 
-be the final state of that flag.
+Get Running in 5 Minutes
+------------------------
 
-Note that the script uses ``stderr`` whenever possible, only sending the DotBot 
-configuration file to ``stdout``. The reasons were not well thought out, but 
-hey, it's a feature.
+```bash
+  wget https://raw.githubusercontent.com/Aviator45003/easy-dotfiles/master/easy_setup.sh
```
Comment (anishathalye) [inline README.md]: I think `curl` is installed by default on OS X, but not `wget`. Could we instead `curl -fsSL` it?

Also, would it be worth rewriting this to `curl ... | sh`? One less line to copy and paste.

**Comment by anishathalye** [comment]
Changes look pretty good to me.

Would a screenshot be good to add? Or even a gif or something?

File: README.md (hunk 1 above)
Comment (Vaelatern) [inline README.md]: Confirmed, wget is not installed by default on Mac OS X.

**Comment by Vaelatern** [comment]
Change the wget to a curl and I'll merge it.

Appropriate flags are -f (for don't return an html error), -s (for silent running), -S (for showing errors even though we are silent). I don't think -L is required, but it does not need to be excluded if you like the idea (following redirects to new places. Perhaps github is finicky about this).

File: README.md (hunk 1 above)
Comment (joshblum) [inline README.md]: fixed, but we can't pipe to shell

**Comment by joshblum** [comment]
Fixed to use curl instead of wget but I don't think we can do `| sh`, since doing so breaks the script (requires user input). See: http://stackoverflow.com/questions/2740868/difference-between-piping-a-file-to-sh-and-calling-a-shell-file

Unless anyone knows a workaround? Command is still copy pasteable. Should we cleanup the file after install?

**Comment by joshblum** [comment]
@anishathalye screenshot could be cool. @Aviator45003 what do you think? 

We can add it to the repo or make github be the CDN via: http://solutionoptimist.com/2013/12/28/awesome-github-tricks/

**Comment by joshblum** [comment]
screen shot 2015-06-11 at 12 33 23 pm

**Comment by Vaelatern** [comment]
A screenshot or gif sounds great. Let's use git as the CDN (nobody who clones wants to see the image in the repo). I am partial to a gif.
Sencondary to getting the readme working.

**Comment by Vaelatern** [comment]
Merged. as of this point.

**Comment by anishathalye** [comment]
Oh, no screenshot in this commit?

**Comment by joshblum** [comment]
cool :)

Let's wait until after the name change, but I would prefer someone else does it. Gif would be cooler but also more to maintain if the script changes..

**Comment by Vaelatern** [comment]
Do people find a project more approachable if there is a screenshot?

I can do it, as well as the gif.

**Comment by Vaelatern** [comment]
@anishathalye, it's a working copy. The screenshot is extra. Right now, the README as merged is better than the old one. That means it's acceptable. We can add a screenshot later.

**Comment by joshblum** [comment]
I think they would, its a nice way to see if the script is what you want
without downloading/running anything.

**Comment by anishathalye** [comment]
Good point Toyam. The new readme is definitely an improvement. We can add the screenshot when we have time.
Also +1 about the name change.

**Comment by joshblum** [comment]
https://github.com/chjj/ttystudio might be useful for making a gif of the install instead of a screenshot

**Comment by Vaelatern** [comment]
Right now I'm dealing with the fact that I don't have a good terminal emulator font (it can't show 'm', for example). If I could take a gif of the console, that would be great. And with that program, I will!

**Comment by joshblum** [comment]
output

I don't think the font looks too bad on this..

Alternatively:
https://showterm.io/
https://github.com/icholy/ttygif

**Pull request closed without merging**